README.md

```diff
@@ -110,12 +120,9 @@ Test geocolor installation
     pytest -m integration $GEOIPS_PACKAGES_DIR/geocolor
 
     # Individual direct test calls, for reference
-    $GEOIPS_PACKAGES_DIR/geocolor/tests/scripts/abi_global.sh
-    $GEOIPS_PACKAGES_DIR/geocolor/tests/scripts/abi.sh
-    $GEOIPS_PACKAGES_DIR/geocolor/tests/scripts/ahi.sh
     $GEOIPS_PACKAGES_DIR/geocolor/tests/scripts/ami.sh
     $GEOIPS_PACKAGES_DIR/geocolor/tests/scripts/fci.sh
     $GEOIPS_PACKAGES_DIR/geocolor/tests/scripts/goes_east.sh
     $GEOIPS_PACKAGES_DIR/geocolor/tests/scripts/goes_west.sh
```

Comment:
Trying to consolidate test scripts and reduce duplication.  Remaining test scripts will all have tiny sector image outputs, no more high res full disk test outputs.

I am guessing we will want to archive the existing very large geocolor repo, and create a completely new one to clear out the large history once we are happy with the layout of our test scripts / outputs.